doc/mongoc_bulk_operation_set_client.rst

```diff
@@ -0,0 +1,27 @@
+:man_page: mongoc_bulk_operation_set_client
+
+mongoc_bulk_operation_set_client()
+==================================
+
+Synopsis
+--------
+
+.. code-block:: c
+
+  void
+  mongoc_bulk_operation_set_client (mongoc_bulk_operation_t *bulk, void *client);
```

Comment:
`void *client` may be regrettable, but it's already part of the public ABI. Should the documentation be consistent, or would you rather see `mongoc_client_t *` here?

Reply:
Keeping documentation consistent.